From 18492536e1b4fc7b53245e5f37dec9420ee17203 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Sat, 5 Dec 2020 20:40:44 +0530
Subject: [PATCH 11/40] Revert "ANDROID: kbuild: do not merge .section..* into
 .section in modules"

This reverts commit ed3e9172a9ebad443679e66d364b7cd21b2590da.
---
 scripts/module-lto.lds | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/scripts/module-lto.lds b/scripts/module-lto.lds
index 5ba0e9461e13..f5ee544a877d 100644
--- a/scripts/module-lto.lds
+++ b/scripts/module-lto.lds
@@ -12,11 +12,11 @@ SECTIONS {
 		*(.eh_frame)
 	}
 
-	.bss : { *(.bss .bss.[0-9a-zA-Z_]*) }
-	.data : { *(.data .data.[0-9a-zA-Z_]*) }
-	.rela.data : { *(.rela.data .rela.data.[0-9a-zA-Z_]*) }
-	.rela.rodata : { *(.rela.rodata .rela.rodata.[0-9a-zA-Z_]*) }
-	.rela.text : { *(.rela.text .rela.text.[0-9a-zA-Z_]*) }
-	.rodata : { *(.rodata .rodata.[0-9a-zA-Z_]*) }
-	.text : { *(.text .text.[0-9a-zA-Z_]*) }
+	.bss : { *(.bss .bss[.0-9a-zA-Z_]*) }
+	.data : { *(.data .data[.0-9a-zA-Z_]*) }
+	.rela.data : { *(.rela.data .rela.data[.0-9a-zA-Z_]*) }
+	.rela.rodata : { *(.rela.rodata .rela.rodata[.0-9a-zA-Z_]*) }
+	.rela.text : { *(.rela.text .rela.text[.0-9a-zA-Z_]*) }
+	.rodata : { *(.rodata .rodata[.0-9a-zA-Z_]*) }
+	.text : { *(.text .text[.0-9a-zA-Z_]*) }
 }
-- 
2.17.1

